docker/syncer/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM debian:bullseye
 RUN apt-get update
-RUN DEBIAN_FRONTEND=noninteractive apt-get --yes install emacs-nox git make mercurial ruby
+RUN DEBIAN_FRONTEND=noninteractive apt-get --yes install locales emacs-nox curl git make mercurial ruby ruby-erubi jq
+RUN curl -ssLO https://github.com/johnkerl/miller/releases/download/v6.8.0/miller-6.8.0-linux-amd64.deb \
```

Comment:
Is there any way to do this without manually downloading a `deb` file?

Reply:
Possibly on the next debian version. `miller` is already packaged in debian, but I was running into an issue that had already been resolved in a later version, so I just updated the package itself for expediency.

Reply:
Looks like it's in the new Debian release. You should feel free to depend on that.

```
apt-cache search miller | grep ^miller
miller - name-indexed data processing tool
```

Reply:
Fixed by rebasing on top of #8634